wire-ios-request-strategy/Sources/Request Strategies/User Clients/ResetSessionRequestStrategy.swift

```diff
@@ -59,18 +59,14 @@ extension ResetSessionRequestStrategy: KeyPathObjectSyncTranscoder {
         // Enter groups to enable waiting for message sending to complete in tests
         let groups = managedObjectContext.enterAllGroupsExceptSecondary()
         Task {
-            let result = await messageSender.sendMessage(message: message)
-
-            switch result {
-            case .success:
-                managedObjectContext.performAndWait {
+            do {
+                try await messageSender.sendMessage(message: message)
+                await managedObjectContext.perform {
                     userClient.resolveDecryptionFailedSystemMessages()
                 }
-            case .failure:
-                break
-            }
+            } catch { }
```

Comment:
should we log the error here? If no try?

Reply:
Done